src/Microsoft.VisualStudio.Composition/RuntimeComposition.cs

```diff
@@ -279,10 +284,13 @@ public class RuntimePart : IEquatable<RuntimePart>
 
             public MethodBase? ImportingConstructorOrFactoryMethod => this.ImportingConstructorOrFactoryMethodRef?.MethodBase;
 
+            [MessagePackFormatter(typeof(MessagePackCollectionFormatter<RuntimeImport>))]
```

Comment:
Nit: Similar question to above but I didn't think IReadOnlyList required a custom formatter.